### DISCUSSION TITLE
How to connect to a UDP only VPN offered by the website itself?

### DISCUSSION THREAD

**Discussion by ghost** [SoftEtherVPN/SoftEtherVPN#2068] [Q&A]
hi there,
I am using the current build available from today 2024-10-14 with Public Relay Plugin.
I can use the plugin and its working but I want to know how can I connect to a vpn with UDP only eg from yesterday show list?
I mean I know DDNS and UDP port it worked yesterday from the list over the plugin, I noted it down for later use.

I am not able to setup this for manual connection. so where is the possibilty to get back to this connection?
Is there a way to add or save favorites to that Plugin List or something like this?

**Comment by chipitsine** [comment]
sorry, no idea what is "Public Relay Plugin".
I suggest you to ask author of that plugin

**Reply by ghost** [reply]
clearly a featured product/part of softether. ;)
https://www.vpngate.net/en/howto_softether.aspx#windows
gfx screen
https://www.vpngate.net/images/ss/softether_win/en/05.jpg

**Reply by chipitsine** [reply]
usually it is called vpngate plugin.

ok, it is clear now. I doubt if any of vpngate developer is on this github tracker. if they are, they are welcome to support customers
